test/powershell/Modules/Microsoft.PowerShell.Utility/Write-Debug.Tests.ps1

```diff
@@ -0,0 +1,17 @@
+Describe "Write-Debug tests" -Tags "CI" {
+    It "Should not have added line breaks" {
+        $text = "0123456789"
+        while ($text.Length -lt [Console]::WindowWidth) {
+            $text += $text
+        }
+        $origDebugPref = $DebugPreference
+        $DebugPreference = "Continue"
+        try {
+            $out = Write-Debug $text *>&1
```

Comment:
Maybe use exactly "2>&1"

Reply:
Debug isn’t stream number 2 and it seems you can only redirect debug if you redirect all streams

Reply:
It seems works:
```powershell
PS> $a=Write-Verbose "ext " -Verbose 4>&1
PS> $a
VERBOSE: ext
PS> $b=Write-Debug "qwerty " -Verbose 5>&1
PS> $b
PS> $b=Write-Debug "qwerty " -Debug 5>&1
Confirm
Continue with this operation?
[Y] Yes  [A] Yes to All  [H] Halt Command  [S] Suspend  [?] Help (default is "Y"): y
PS C:\Users\sie> $b

DEBUG: qwerty
```

Reply:
Maybe I did something wrong when I tried it.  Will update.